tox.ini

```diff
@@ -43,6 +43,7 @@ deps =
     Sphinx
 
 [pep8]
+ignore = E123,E266
```

Comment:
Ahh I get it.

I'm not sure about the "more error-prone" bit, but I'm in agreement with you that this isn't a problem we should solve with more code.

Check out my update to #719. I simply move the LICENSE after the "You don't need to do anything, just keep hitting enter..." statement and just let it get printed after we set the expectations with the demo user that they can just hit enter.